source/braille.py

```diff
@@ -3284,7 +3291,9 @@ class BrailleDisplayDriver(driverHandler.Driver):
 	containing a BrailleDisplayDriver class which inherits from this base class.
 
 	At a minimum, drivers must set L{name} and L{description} and override the L{check} method.
-	To display braille, L{numCells} and L{display} must be implemented.
+	To display braille, L{display} must be implemented.
+	For a single line display, L{numCells} must be implemented.
+	For a multi line display, L{numRows} and L{numCols} must be implemented.
```

Comment:
Could you please update this docstring to use [Sphinx style docstrings](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html) per our [coding standards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md#docstrings)?

Partial fix:
```suggestion
	To display braille, :meth:`display` must be implemented.
	For a single line display, :attr:`numCells` must be implemented.
	For a multi line display, :attr:`numRows` and :attr:`numCols` must be implemented.
```

Resources:
* [reStructuredText Primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html)
* [Sphynx' custom reStructuredText Directives](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html)
* [Sphynx' Python Domain](https://www.sphinx-doc.org/en/master/usage/domains/python.html)

Reply:
I'll try to do that later on.